app/controllers/characters_controller.rb

```diff
@@ -61,7 +61,6 @@ def create
 
   def show
     @page_title = @character.name
-    @posts = posts_from_relation(@character.recent_posts)
```

Comment:
Is this not used in the html?

Reply:
(Yes, but only in some cases, so I embedded it directly)

Reply:
ah, I didn't get to that yet

Reply:
Test expectations wanted it to be assigned anyway, so I've moved it back with an if!